### DISCUSSION TITLE
Face detection configuration options, is it possible to recognize 15x15px faces?

### DISCUSSION THREAD

**Discussion by Howiser1** [photoprism/photoprism#2062] [General]
Ubuntu 21.10 server x64
PhotoPrism version 220121-2b4c8e1f-Linux-x86_64

Mostly interested in this setting:  --face-size
Firstly, the lowest value is 20 (that I can set).  This is documented no where.

This setting would not take effect even from the docker-compose.yml
PHOTOPRISM_FACE_SIZE: "15"
This has no effect -- Face-size in the config is still 50
BECAUSE it will not expect anything lower than 20.

AFTER trying many different command line permutations and with great frustration:  THIS FINALLY WORKED

docker-compose exec photoprism photoprism --face-size 40 **config**

NOTICE: the **config** at the end.  IT MUST be there.  This is documented no where!
one example in your Config documentation could have saved HOURS of frustration!

**Comment by lastzero** [comment]
Note that sponsors receive direct support via email. That would have saved you hours of frustration! Just saying.

Before you rant on, can you show us a few examples of 15x15 faces that you can clearly identify?

I'll give you an easy one: 

![foto_no_exif](https://user-images.githubusercontent.com/301686/155302123-22e4c110-6cc7-4b09-a71e-5f40d062d4c1.jpg)

Who is this?

**Reply by lastzero** [reply]
Today is your lucky day, because instead of taking a break because I look like a zombie (or improving multi-user support), I updated the documentation for you:
https://docs.photoprism.app/getting-started/config-options/#facial-recognition

A contributor like YOU could have easily added this, which would have saved us time for other work. The source code is public, you can easily look it up. No excuses.

**Reply by lastzero** [reply]
Okay, so that was Boris Johnson. If humans can't recognize him like this, 15 pixels is probably too small for a machine. It would not be helpful to allow such settings.

That being said, our documentation clearly states that these settings are for advanced users/developers only. I will mark the question as answered since the documentation and command help have been improved and we have not received any new feedback.

**Comment by Coloradohusky** [comment]
Note that for those using `photoprism:preview`, you must suffix `--face-size 40`, or whatever other config value you wish to change, with `show config` instead of simply `config`, due to the preview version nesting `config` under `show`.

Also, added an example for changing config settings here (https://github.com/photoprism/photoprism-docs/pull/112).
